hardware_enablement/kmm-release-notes.adoc

```diff
@@ -28,4 +27,297 @@ toc::[]
+
+** *Cause*: Creating a `PreflightValidationOCP` custom resource (CR), with a 'kernelVersion' flag that is set to 'tesdt', causes the Kernel Module Management (KMM) Operator to generate a panic runtime error.
+
+** *Consequence*: Entering invalid kernel versions causes the KMM Operator to panic.
+
+** *Fix*: A webhook - a method for one application to automatically send real-time data to another application when a specific event occurs - has been added to the `PreflightValidationOCP` CR.
+
+** *Result:* The `PreflightValidationOCP` CR with invalid kernel versions can no longer be applied to the cluster, therefore, preventing the operator from generating a panic runtime error.
+
+// MGMT-20866
+* The `PreFflightValidationOCP` custom resource, with a `kernelVersion` flag that is different that the one of the cluster, does not work.
+
+** *Cause*: Creating a `PreflightValidationOCP` custom resource (CR), with a 'kernelVersion' flag that is different from the one of the cluster, does not work.
+
+** *Consequence*: The Kernel Module Management (KMM) Operator is unable to find the Driver Toolkit (DTK) input image for the new kernel version.
+
+** *Fix*: Users should use the `PreflightValidationOCP` CR and explicitly set the 'dtkImage' field in the CR.
+
+** *Result:* Using the fields `kernelVersion` and `dtkImage` the feature can build installed modules for target {product-title} versions.
+
+// MGMT-20888
+* The KMM Operator version 2.4 documentation is updated with `PreflightValidationOCP` information.
+
+** *Cause*: Previously, when creating an `PreflightValidationOCP` CR, you were required to supply the release-image. This has now changed and you need to set the `kernelVersion` the `dtkImage` fields.
+
+** *Consequence*: The documentation was outdated and required an update.
+
+** *Fix*: The documentation is updated with the new support details.
+
+** *Result:* The KMM preflight feature is documented as expected.
+
+// MGMT-20892
+* Using imageRepoSecret in conjunction with DTK as imagestream gets authorization required error.
+
+** *Cause*: On the Kernel Module Management (KMM) Operator, when you set the `imageRepoSecret` in the KMM module, and the build's resulting container image is defined to be stored in the cluster's internal registry, the build will fail to push the final image and generate an `authorization required` error.
+
+** *Consequence*: The KMM Operator does not work as expected.
+
+** *Fix*: When the `imageRepoSecret` is user-defined, it is used as both a pull and push secret by the build process. In order to support using the cluster's internal regsitry, the auth token for that registry needs to be added to the `imageRepoSecret`. The token can be obtained from the "build" service account of the KMM module's namespace.
+
+** *Result:* The KMM Operator works as expected.
+
+// test
+
+=== Known issues
+// MGMT-20453 changed security settings in jira
+* The `ModuleUnloaded` event does not appear when a module is Unloaded.
```

Comment:
Suggestion based on the assumption that unloaded is a state, if incorrect, please use the noun form
```suggestion
* The `ModuleUnloaded` event does not appear when a module is `Unloaded`.
```